proto/cel/expr/conformance/env_config.proto

```diff
@@ -0,0 +1,159 @@
+
+option cc_enable_arenas = true;
+option go_package = "cel.dev/expr/conformance";
+option java_multiple_files = true;
+option java_outer_classname = "EnvironmentProto";
+option java_package = "cel.dev.expr.conformance";
+
+// Representation of a CEL Environment, defining what features and extensions
+// are available for conformance testing.
+message Environment {
+  // Name of the environment
+  string name = 1;
+
+  // Description for the current environment
+  string description = 2;
+
+  // Sets the namespace (container) for the expression.
+  // This is used to simplify resolution: e.g. with container
+  // `google.rpc.context`, an identifier of
+  // `google.rpc.context.AttributeContext` could be referred to simply as
+  // `AttributeContext` in the CEL expression.
+  string container = 3;
+
+  // Import represents a type name that will be abbreviated by its simple name
+  // using the cel.Abbrevs() option.
+  message Import {
+    // Qualified type name which will be abbreviated
+    string name = 1;
+  }
+
+  // List of abbreviations to be added to the CEL environment
+  repeated Import imports = 4;
+
+  // Set of options to subset a subsettable library
+  LibrarySubset stdlib = 5;
+
+  // List of extensions to enable in the CEL environment.
+  repeated Extension extensions = 6;
+
+  // ContextVariable represents a message type to be made available as a
+  // context variable to the CEL environment.
+  message ContextVariable {
+    // Fully qualified type name of the context proto.
+    string type_name = 1;
+  }
+
+  // If set, adds a context declaration from a proto message.
+  //
+  // Context messages have all of their top-level fields available as variables
+  // in the type checker.
+  repeated ContextVariable context_variables = 7;
+
+  // List of declarations to be configured in the CEL environment.
+  repeated cel.expr.Decl declarations = 8;
+
+  // List of validators for validating the parsed ast.
+  repeated Validator validators = 9;
+
+  // List of feature flags to be enabled or disabled.
+  repeated Feature features = 10;
+
+  // Disables including the declarations from the standard CEL environment.
+  //
+  // NOTE: Do not disable the standard CEL declarations unless you are aware of
+  // the implications and have discussed your use case on cel-discuss@
+  // or with the members of the cel-governance-team@
+  bool disable_standard_cel_declarations = 11;
+
+  // If provided, uses the provided FileDescriptorSet to extend types available
+  // the CEL expression. All "well-known" protobuf messages (google.protobuf.*)
+  // are known to the CEL compiler, but all others must be provided for type
+  // checking.
+  google.protobuf.FileDescriptorSet message_type_extension = 12;
+
+  // When macro call tracking is enabled, the resulting SourceInfo in the
+  // CheckedExpr will contain a collection of expressions representing the
+  // function calls which were replaced by macros.
+  bool enable_macro_call_tracking = 13;
```

Comment:
We can mark this deprecated and mention that the preferred representation is `Feature{name: "cel.feature.macro_call_tracking", enabled: true}`. Removal can happen later though.